indigo_api/static/xsl/html_akn.xsl

```diff
@@ -55,17 +55,41 @@
     </xsl:attribute>
   </xsl:template>
 
-  <xsl:template match="a:article">
+  <!-- containers with headings on the next line -->
+  <xsl:template name="container-1">
```

Comment:
Minor: `container` has a specific meaning in AKN. Perhaps `hier-heading-next-line` or similar is better?